CONTRIBUTING.md

```diff
@@ -21,6 +21,14 @@ or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any addi
 ## Recommended Tools
 * Visual Studio and Visual Studio Code https://visualstudio.microsoft.com/
 
+## Git
+
+### Conventions
+TODO
```

Comment:
Some things we can cover here: 
* Merging or Rebasing to update branches
* Pull Request merge strategy (Merge, or squash merge, or rebase and merge I think are the values we have set to allowed)
* Commit messages
* PR Size in terms of easiness to reason about.

Reply:
This is a subjective take, so feel free to disagree, but I'm personally not a fan of "conventional commit" formatting (e.g. `refactor`, `docs`, `style`, etc.). I generally prefer to see labels on PRs used for that. Rather than assigning a `<type>` to a commit the main thing I want to see is just a description of what changed.

These [guidelines](https://cbea.ms/git-commit/#seven-rules) are usually enough for me.